pymc/distributions/AR1.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>@ricardoV94 any chance this error happens because I use a for loop in the logp instead of scan</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5388/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>